lib/bolt/inventory.rb

```diff
@@ -53,11 +54,23 @@ def self.from_config(config)
         data = Bolt::Util.read_config_file(config.inventoryfile, config.default_inventoryfile, 'inventory')
       end
 
-      inventory = new(data, config)
+      inventory = create_version(data, config)
       inventory.validate
       inventory
     end
 
+    def self.create_version(data, config)
+      version = (data || {}).delete('version') { 1 }
+      case version
+      when 1
```

Comment:
This should probably follow semantic versioning unless you want *every* update to be breaking.

It would make sense to be able to fix typos and/or add fields without having to bump the version number.

Reply:
We currently only intend to version this file for breaking changes. We've added many features to version `1` without worrying about incrementing or noting the minor version number. This aligns with other file formats in the puppet world.

Reply:
The problem we're trying to solve is to allow one version of bolt to support multiple different incompatible versions of the inventory file format. By specifying the `version` bolt knows which implementation to use to load the file. That is a different problem than semver is intended to solve which is choosing a compatible version of a library to install when resolving dependencies.